pkg/controller/controller.go

```diff
@@ -69,14 +71,26 @@ func init() {
 	utilruntime.Must(v1alpha1.AddToScheme(scheme.Scheme))
 }
 
-var (
-	v1alpha1ClusterGVR  = v1alpha1.SchemeGroupVersion.WithResource(v1alpha1.SpiceDBClusterResourceName)
-	OwnedFactoryKey     = typed.NewFactoryKey(v1alpha1.SpiceDBClusterResourceName, "local", "unfiltered")
-	DependentFactoryKey = typed.NewFactoryKey(v1alpha1.SpiceDBClusterResourceName, "local", "dependents")
-)
+var v1alpha1ClusterGVR = v1alpha1.SchemeGroupVersion.WithResource(v1alpha1.SpiceDBClusterResourceName)
+
+func localClusterNamespace(ns string) string {
+	if ns == "" {
+		return "local"
```

Comment:
Where does this come from?

Reply:
it's just a part of the key used to locate the right cache. I use "local" as a convention because it's part of the same cluster the controller is running on, and controllers can also start watches of remote clusters.